packages/auth/src/accessToken/service.ts

```diff
@@ -72,16 +90,41 @@ async function introspect(
   }
 }
 
-async function revoke(
-  deps: ServiceDependencies,
-  id: string
-): Promise<Error | undefined> {
+async function revoke(deps: ServiceDependencies, id: string): Promise<void> {
   const token = await AccessToken.query(deps.knex).findById(id)
   if (token) {
-    if (!isTokenExpired(token)) {
-      await token.$query(deps.knex).patch({ expiresIn: 1 })
+    await token.$query(deps.knex).delete()
+  }
+}
+
+async function rotate(
+  deps: ServiceDependencies,
+  managementId: string
+): Promise<Rotation> {
+  let token = await AccessToken.query(deps.knex).findOne({ managementId })
+  if (token) {
+    await token.$query(deps.knex).delete()
+    const newManagementId = uuid()
```

Comment:
Is there a reason you define that separately? Why not just call the `uuid` function in line 112?

Reply:
Originally I was returning the value but since I don't have to now I can just call the function